### DISCUSSION TITLE
Changing the default Ride Name?

### DISCUSSION THREAD

**Discussion by FlyingThunder** [OpenRCT2/OpenRCT2#20697] [Feature requests]
What a silly idea, but i recently noticed that it would be kind of weird to go into a theme park and see a ride called "Wooden Rollercoaster 1", even though there is only one Wooden Rollercoaster. Can someone roughly point me at where the increments for the ride numbering is decided, so i can fork and change it from Ride1->Ride2->RideN to Ride->Ride2->RideN? Im one of those guys who renames every Ride, even stalls so theres no "1" in them 😁

**Comment by ZehMatt** [comment]
https://github.com/OpenRCT2/OpenRCT2/blob/ed32fd2567059aa99505052d74a93f18297d3bca/src/openrct2/ride/Ride.cpp#L4307

**Comment by MASHzj** [comment]
I always remove those 1's too 😄 unless when I already know that I will build the ride or stall multiple times. So I wouldn't mind if this would change in the default OpenRCT2 🙂.

**Comment by ocalhoun6** [comment]
Maybe this would be a good job for a plugin?

**Reply by Broxzier** [reply]
Nah, just change the default. It's stilly that a 1 is appended to the first ride by default.

**Comment by 733737** [comment]
I would be happy if this were implemented in the main branch. I don't think this should be relegated to a fork, it does not harm anything.
